Update RHEL worker section with UBI and envdir (#1928)
hashicorp/terraform-website: content/source/docs/enterprise/install/installer.html.md

![Terraform Enterprise docker image](./assets/tfe-docker-image.png)

### Requirements

- The base image must be `ubuntu:bionic` or an [offical RHEL7 image](https://catalog.redhat.com/software/containers/search?p=1&vendor_name=Red%20Hat%2C%20Inc.&rows=60&product_listings_names=Red%20Hat%20Universal%20Base%20Image%207|Red%20Hat%20Enterprise%20Linux%207&build_categories_list=Base%20Image)
  (e.g., `registry.access.redhat.com/ubi7/ubi-minimal`).

> - The image must exist on the Terraform Enterprise host. It can be added by running `docker pull` from a local registry or any other similar method.

Reviewer: ```suggestion
- The image must exist on the Terraform Enterprise host. You can add it by running `docker pull` from a local registry or any other similar method.
```

- The software packages defined in the examples below must be installed on the image.
- All necessary PEM-encoded CA certificates must be placed within the `/usr/local/share/ca-certificates` directory. Each file added to this directory must end with the `.crt` extension. The CA certificates configured in the [CA Bundle settings](#certificate-authority-ca-bundle) will not be automatically added to this image at runtime.

> - Terraform must not be installed on the image. Terraform Enterprise will take care of that at runtime.

Reviewer: ```suggestion
- Terraform must not be installed on the image. Terraform Enterprise installs Terraform at runtime.
```

#### RHEL 7

> Note that in addition to the packages installed by yum, the Python port
> of the `envdir` tool is also installed as it is used by TFE. If this
> tool is not installed on the alternative worker image, Terraform plan
> and apply operations will fail.

Reviewer: Also - is TFE defined anywhere as an acronym on this page? I noticed that we switch back and forth between writing out Terraform Enterprise and using TFE.  My recommendation is to at the very top, define it at first use like this:  "Terraform Enterprise (TFE)"  and then just use TFE consistently throughout. But we shouldn't use the acronym unless it's defined, even if we think people probably know what it is  :-) 

```suggestion
In addition to the packages that yum installs, Terraform Enterprise also installs the the `envdir` tool Python port. If the `envdir` tool is not installed on the alternative worker image, Terraform plan and apply operations will fail.
```

Author: 💯 ❌ 💯

Author: I'm very much in favour of consistently defining acronyms 👍🏼 

I modified your suggestion to clarify where and why envdir is installed. What do you think?
```suggestion
In addition to the packages that yum installs, the envdir tool Python port is also installed as it is a dependency of the Terraform Build Worker service. If the envdir tool is not installed on the alternative worker image, Terraform runs will fail within Terraform Enterprise.
```

Reviewer: Oh I really like it!! my only final nit is - can we make it not passive? What is doing the installing? In an ideal world, I think we'd want to say something like "In addition to the packages that yum installs, [name of thing] installs the envdir tool Python port because it is a dependency of the Terraform Build Worker service."

Author: Ah, sorry. How about this?

```suggestion
In addition to the packages that yum installs, curl installs the envdir tool Python port because it is a dependency of the Terraform Build Worker service. If the envdir tool is not installed on the alternative worker image, Terraform runs will fail within Terraform Enterprise.
```

Reviewer: love that - merge it!
